src/meta/src/controller/catalog/mod.rs

```diff
@@ -990,18 +990,48 @@ impl CatalogControllerInner {
             .collect())
     }
 
-    /// `list_indexes` return all `CREATED` indexes.
+    /// `list_indexes` return all `CREATED` and `BACKGROUND` indexes.
```

Comment:
I suppose we don't list all indexes here, because there might be some paths which don't handle `foreground + creating` indexes?

Reply:
I didn't quite understand, mainly referring to which path?

Reply:
What I mean is, why don't we just list all indexes? And only created / creating + background.

Reply:
Opened an issue for it: https://github.com/risingwavelabs/risingwave/issues/23031